examples/expressions_explorer/public/editor/expression_editor.tsx

```diff
@@ -16,19 +16,16 @@ interface Props {
 
 export function ExpressionEditor({ value, onChange }: Props) {
   return (
-    <EuiCodeEditor
-      mode="javascript"
-      theme="github"
+    <CodeEditor
```

Comment:
Aside from build errors, it not longer makes sense to use `EuiCodeEditor` when `CodeEditor` is the preferred component.

Open to suggestions on improving this configuration from folks who have more experience using `CodeEditor`.

Reply:
@thompsongl FYI that this change is causing the failing functional test / entire expressionsExplorer page to crash:

http://localhost:5061/dev/app/expressionsExplorer

<img width="921" alt="" src="https://user-images.githubusercontent.com/549407/140821053-195aacd2-e708-43ef-a746-d60e9ad2e62b.png">

Reply:
I didn't think this change was actually part of the app? It's part of `examples/`

Reply:
Should be resolved with 60170a9

Reply:
Nice!